rtm.go

```diff
@@ -8,11 +8,18 @@ import (
 	"time"
 )
 
+const (
+	WEBSOCKET_TIMEOUT_SECONDS = 10 * time.Second
```

Comment:
weboscketTimeoutSeconds, no need to clutter up the exposed API with immutable internal constants.